SingularityUI/app/components/deployDetail/DeployDetail.jsx

```diff
@@ -353,6 +362,7 @@ function mapStateToProps(state, ownProps) {
     deploy: state.api.deploy.data,
     taskHistory: state.api.taskHistoryForDeploy.data,
     isTaskHistoryFetching: state.api.taskHistoryForDeploy.isFetching,
+    group: state.api.deploy.data.deploy && _.first(_.filter(state.api.requestGroups.data, (g) => _.contains(g.requestIds, state.api.deploy.data.deploy.requestId))),
```

Comment:
I'm adding a linting check to prohibit one-letter variable names (See #1192). Might want to change `g` to `groupToCheck`, or something more descriptive.